aws/components/mta/setup.ftl

```diff
@@ -85,6 +86,33 @@
                     [#list topicArns as topicArn ]
                         [#-- notifications must be done through CLI --]
                         [#if deploymentSubsetRequired("epilogue", false) ]
+                            [#local addBash = []]
+                            [#local updBash = []]
+                            [#local delBash = []]
```

Comment:
Lets merge recipient details into a single command that is added to the epilogue script. then this can be a singe variable for looking after the recipient assignemnt